daprdocs/content/en/reference/components-reference/supported-bindings/kafka.md

```diff
@@ -74,6 +74,10 @@ This component supports **output binding** with the following operations:
 
 - `create`
 
+## Authentication
+
+Kafka supports a variety of authentication schemes and Dapr supports several: SASL password, mTLS, OIDC/OAuth2. [Learn more about Kafka's authentication method for both bindings and pub/sub]({{< ref "setup-apache-kafka.md#authentication" >}}).
```

Comment:
I would be more specific like this

Kafka supports a variety of authentication schemes and Dapr supports several: SASL password, mTLS, OIDC/OAuth2. [Learn more about Kafka's authentication method for both the Kafka binding and Kafka pub/sub components]({{< ref "setup-apache-kafka.md#authentication" >}}).